res_company_category/views/view_res_company_category.xml

```diff
@@ -6,12 +6,12 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 <odoo>
 
-
     <record id="view_res_company_category_tree" model="ir.ui.view">
+        <field name="name">res.company.category.tree</field>
```

Comment:
No need to add the name here. Odoo automatically does the job.

Reply:
I misread the doc ! changes made